draft-ietf-quic-datagram.md

```diff
@@ -138,6 +138,10 @@ use them. It is RECOMMENDED to send the value 65536 in the
 max_datagram_frame_size transport parameter as that indicates to the peer that
 this endpoint will accept any DATAGRAM frame that fits inside a QUIC packet.
 
+The max_datagram_frame_size transport parameter is a unidirectional limit and
+indication of support of DATAGRAM frames. It is allowable for a protocol to
```

Comment:
nit: `s/for a protocol/for an application protocol/`

Reply:
Actually we can make this normative:

`Application protocols that use DATAGRAM frames MAY choose to only negotiate and use them in a single direction.`

Reply:
Sure. Let me update that now.

Reply:
Thanks!